src/lib/stitching/causality/stitching.ts

```diff
@@ -22,9 +28,57 @@ export const causalityStitchingEnvironment = ({
       extend type AuctionsLotStanding {
         saleArtwork: SaleArtwork
       }
+
+      extend type AuctionsLotState {
+        floorSellingPrice2: Money
+      }
     `,
 
     resolvers: {
+      AuctionsLotState: {
+        floorSellingPrice2: {
+          fragment: gql`
+            ... on AuctionsLotState {
+              internalID
+              floorSellingPrice
+            }
+          `,
+          type: Money,
+          resolve: async (
+            { internalID, floorSellingPrice },
+            _args,
+            context,
+            info
+          ) => {
+            const { currency } = await context.saleArtworkRootLoader(internalID)
+            const { units: minor } = floorSellingPrice
+            const major = await moneyMajorResolver(
+              { cents: minor, currency },
+              {},
+              context,
+              info
+            )
+
+            // return { cents: minor, currency }
```

Comment:
I was hoping there is an easy way to keep resolving this node as a `Money` type.

Reply:
Paging @alloy and curious if you have insights or recommendations.